pkgs/cupertino_http/pubspec.yaml

```diff
@@ -4,10 +4,11 @@ description: >-
   A macOS/iOS Flutter plugin that provides access to the Foundation URL
   Loading System.
 repository: https://github.com/dart-lang/http/tree/master/pkgs/cupertino_http
+publish_to: no
```

Comment:
```suggestion
publish_to: none
```

Reply:
The `publish_to` directive was there by accident.